src/Utilities/ProgramUtils.tsx

```diff
@@ -15,7 +15,7 @@
 const extractGameNameFromProgram = pc => {
   // extract game name, opting for a longer name first, and then a shorter name after
   // if a longer name is not able to be matched
-  let gnMatch = pc.match(/game\s+([A-Z][A-Za-z0-9_]+)|([A-Z])/);
+  let gnMatch = removeComments(pc).match(/game\s+([A-Z][A-Za-z0-9_]+|[A-Z])/);
```

Comment:
This was the latest problem that was fixed. The unnamed captures I was using before, combined with the choice of alternation at the end (`|([A-Z])...`) resulted in inconsistent matching for games of one or more characters, ex. `game G` and `game GG...` matched at different capture points. I unified both of these together under the same capture, and properly placed the alternation inside, so both use the same capture point.